### DISCUSSION TITLE
posts/announcing-project-polyclimate/

### DISCUSSION THREAD

**Discussion by giscus** [polyclimate/polyclimate.org#2] [Announcements]
# posts/announcing-project-polyclimate/

A place to think about collaborative climate science.

https://polyclimate.org/posts/announcing-project-polyclimate/

**Comment by rgieseke** [comment]
Two projects which give me hope that collaborative science in the open can work are
https://github.com/openclimatedata/ipcc-bibtex/ and https://github.com/openclimatedata/globalwarmingpotentials/

BibTeX citations for IPCC reports (benefits for contributors obvious) and GWPs which are shared in different projects and really shouldn't be typed up again and again from PDFs.

**Comment by mikapfl** [comment]
Nice initiative, I also think we need a lot less duplication of work, and more open collaboration on climate datasets etc.

In the stuff we already tried with primap-hist and the surrounding data reading effort, we found it is easy to put code and small data like the GWP data (nice work!) on github and collaboration in a small community happens naturally. Of course, it is sometimes hard to know what exists, and we end up duplicating work more often than necessary, so there is room for improvement.
But the things which are not smooth at all, yet, have to do with larger data sets. When you hit the limits of what github is willing to host, but still want to collaborate in the open, I found we were out of options. We installed a testing server for gin, a git-style data management platform, and our efforts to develop one dataset there are here: https://gin.hemio.de/jguetschow/UNFCCC_non-AnnexI_data
This kinda works, but honestly, we don't really want to do data hosting, certainly not for the dozen or so datasets we would want to work on ourselves. 

The crazy thing is, nobody seems interested in setting up an infrastructure like https://gin.g-node.org/ but for climate policy analysis data. All the institutions just build tools which work for employees of their institution and maybe 2 selected outside collaborators only. For us, it is a bit of a blocker.

**Reply by rgieseke** [reply]
Absolutely, this is a great example. The challenge there seems to be both technical and social, it doesn't look trivial to setup everything to be able to contribute and help with PDF reading.

It's also an important point whether GitHub is a suitable platform. At least for coordination I think it is, but (larger) data hosting is of course a challenge.

(Also mirrored here: https://github.com/JGuetschow/UNFCCC_non-AnnexI_data)

**Reply by mikapfl** [reply]
Note that the mirroring on github doesn't contain the data itself and the results, only the code. )-:

**Comment by rgieseke** [comment]
Very interesting read on using GitHub Actions to manage and run a collaborative project (even with non-devs and not only for code).

https://github.blog/2022-07-06-managing-a-game-dev-community/
